libs/blocks/section-metadata/section-metadata.css

```diff
@@ -145,7 +145,7 @@ main>.section[class*='-up']>.content {
   padding-right: var(--grid-margins-width);
 }
 
-.section.five-up {
+.section.five-up :not(.icon-block.inline) {
```

Comment:
We probably shouldn't add block specific selectors in section-metadata.css even in the :not() case. 
Can we make this an override in the icon-block.css rather?